Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -1,12 +1,5 @@
-<<<<<<< HEAD:Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt
```

Comment:
If you want a cleaner git history, you could avoid introducing and reverting this git conflict stuff from 6d7c1179f711cab78601543fcbb2ca3393826cfd and cfb709574e53e0f50310d36fbd6b8774c57b233c

Reply:
It's going to be quite some work to clean that up. I would have to rebuild the commits one by one. Or I could just squash everything together. It's your call.

Reply:
I don't care enough for it that should you spend time on it. 
Your time/effort is more appreciated elsewhere. 
Let's keep the commit history as it is nicely separated into logical changes.